src/flow2supera/reader.py

```diff
@@ -237,6 +237,7 @@ def GetEvent(self, event_index):
             event_interactions = interactions_array[interactions_array['event_id'] == result.true_event_id]
 
             for ixn_idx, ixn in enumerate(event_interactions):
+                if 'target' not in ixn.dtype.fields: continue
```

Comment:
I would keep this change local/ in a separate branch as well. Too specific to particle bomb and probably too specific to the current set of particle bomb files